test/test_oneview_logical_switch_group_facts.py

```diff
@@ -28,8 +28,8 @@
 class TestLogicalSwitchGroupFactsModule(OneViewBaseFactsTest):
     @pytest.fixture(autouse=True)
     def specific_set_up(self, setUp, testing_module):
-        self.PARAMS_GET_ALL = self.EXAMPLES[0]['oneview_logical_switch_group_facts']
-        self.PARAMS_GET_BY_NAME = self.EXAMPLES[4]['oneview_logical_switch_group_facts']
+        self.PARAMS_GET_ALL = {'config': 'config.json', 'state': 'present'}
```

Comment:
Suggestion,
Keep instance variables in small letters or keep these variables outside the class to follow the same style as other test modules.